### PR TITLE
enh(Resource Access): Enable/Disable rule

### PR DESCRIPTION
**Pull request by dmyios**
## Description

Implemented Enable/Disable rule listing action.
[video.webm](https://github.com/centreon/centreon/assets/108675430/48e3b39b-4dfc-4b78-8367-8821ac1e939c)

**Fixes** # MON-37008

## Type of change

- [ ] Patch fixing an issue (non-breaking change)
- [x] New functionality (non-breaking change)
- [ ] Breaking change (patch or feature) that might cause side effects breaking part of the Software

## Target serie

- [ ] 22.04.x
- [ ] 22.10.x
- [ ] 23.04.x
- [ ] 23.10.x
- [x] 24.04.x (master)

#### Community contributors & Centreon team

- [x] I have followed the **coding style guidelines** provided by Centreon
- [x] I have commented my code, especially new **classes**, **functions** or any **legacy code** modified. (***docblock***)
- [x] I have commented my code, especially **hard-to-understand areas** of the PR.
- [x] I have **rebased** my development branch on the base branch (master, maintenance).

### CODE REVIEW

File: centreon/www/front_src/src/ResourceAccessManagement/Listing/Actions/Activate.tsx [hunk 1]
```diff
@@ -0,0 +1,75 @@
...
+        backgroundColor: theme.palette.success.main,
+        opacity: 1
+      },
+      color: theme.palette.common.white
+    }
+  },
+  '& .MuiSwitch-thumb': {
+    backgroundColor: theme.palette.common.white
+  }
+}));
```
Comment (Med1233) [inline centreon/www/front_src/src/ResourceAccessManagement/Listing/Actions/Activate.tsx:31]: move it to separated file

Comment (dmyios) [reply centreon/www/front_src/src/ResourceAccessManagement/Listing/Actions/Activate.tsx]: done

**Review by Med1233 — APPROVED** [review]

**Review by Noha-ElAbrouki — APPROVED** [review]

File: centreon/www/front_src/src/ResourceAccessManagement/Listing/Actions/Activate.styles.ts
Comment (Thebarda) [inline centreon/www/front_src/src/ResourceAccessManagement/Listing/Actions/Activate.styles.ts:1]: I't would be a great deal to move the switch into Centreon UI, what do you think?

Comment (dmyios) [reply centreon/www/front_src/src/ResourceAccessManagement/Listing/Actions/Activate.styles.ts:1]: yep, I'll do it in my next PR ;)

**Pull request merged**